federation/develop/develop.sh

```diff
@@ -69,7 +69,7 @@ function dirty_sha() {
   mkdir -p "${tmp_objects_dir}"
   cp "${index}" "${tmp_index}"
 
-  local -r files=$(git ls-files -m -o -d --exclude-standard)
+  local -r files="$(git ls-files $(git rev-parse --show-toplevel) -m -o -d --exclude-standard)"
```

Comment:
To be honest, I dont have much idea about this change, but its good as long as it works :)
Will be great if you can add a comment which might help anyone reading this code.
Something along the lines of what you have as the PR description should be good.